ecommerce_payfort/templates/payment/payfort.html

```diff
@@ -29,8 +29,34 @@
         </div>
         {# This adds the header for the page. #}
         {% include 'edx/partials/_student_navbar.html' %}
+        
+        
         <form action="{{ payment_result_url }}" class="paymentWidgets" data-brands="{{ brands }}">
         </form>
+        
+        {# This form is a placeholder to be filled programmatically from the View class #}
+        {# The values will also include details about the course being bought #}
+        {# Refer to https://paymentservices-reference.payfort.com/docs/api/build/index.html?python#standard-merchant-page-integration-request #}
+
+        <form name="aps_form" action=“https://sbcheckout.payfort.com/FortAPI/paymentPage” method="POST" target="standard_checkout_iframe">
+            <input type=“hidden” name=“service_command” value=“TOKENIZATION”>
```

Comment:
```suggestion
            <input type="hidden" name="service_command" value="TOKENIZATION">
```

Please use regular `"` quotes instead of stylized quotes `“`.